utils/model_uploader/model_listing/pretrained_models_all_versions.json

```diff
@@ -204,5 +204,16 @@
         "description": "This is a neural sparse encoding model: It transfers text into sparse vector, and then extract nonzero index and value to entry and weights. It serves in both ingestion and search."
       }
     }
+  },
+  {
+    "name": "amazon/sentence-highlighting/opensearch-semantic-highlighter-v1",
+    "versions": {
+      "1.0.0": {
+        "format": [
+          "torch_script"
```

Comment:
Do we have ONNX format ?

Reply:
As of P0, we only have torch script format.